io.github.sigmasd.nosleep.yml

```diff
@@ -0,0 +1,39 @@
+app-id: io.github.sigmasd.nosleep
+
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Platform
+
+command: nosleep
+
+finish-args: 
+  - --socket=wayland
+  - --socket=fallback-x11
+  # Without this it errors with `Failed to register: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown`
+  - --socket=session-bus
```

Comment:
this is not alllowed. so need to figure out which one is missing with `--log-session-bus`

Reply:
Maybe its because I'm using libgio to call dbus on the current session ?

here is the log
```
C1: -> org.freedesktop.DBus call org.freedesktop.DBus.Hello at /org/freedesktop/DBus
C2: -> org.freedesktop.DBus fake wildcarded AddMatch for io.github.sigmasd.nosleep
C3: -> org.freedesktop.DBus fake wildcarded AddMatch for org.mpris.MediaPlayer2.io.github.sigmasd.nosleep
C4: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.portal
C5: -> org.freedesktop.DBus fake ListNames
B-1: <- org.freedesktop.DBus return from C1
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C2
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C3
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C4
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C5
C6: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.DBus
C7: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Desktop
C8: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Documents
C9: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.IBus
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C6
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C7
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C8
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C9
*SKIPPED*
C10: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C11: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C12: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C13: -> org.freedesktop.DBus call org.freedesktop.DBus.StartServiceByName at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C10
B-1: <- org.freedesktop.DBus return from C11
B-1: <- org.freedesktop.DBus return from C12
B-1: <- org.freedesktop.DBus return from C13
C14: -> org.freedesktop.DBus call org.freedesktop.DBus.GetNameOwner at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C14
C15: -> :1.79 call org.freedesktop.DBus.Properties.GetAll at /org/freedesktop/portal/desktop
B3898: <- :1.79 return from C15
C16: -> :1.79 call org.freedesktop.portal.Settings.ReadAll at /org/freedesktop/portal/desktop
B3902: <- :1.79 return from C16
C17: -> org.freedesktop.DBus call org.freedesktop.DBus.RequestName at /org/freedesktop/DBus
Filtering message due to arg0 io.sigmasd.nosleep, policy: 0 (required 3)
*HIDDEN* (ping)
B-1: <- org.freedesktop.DBus return from C17
*REWRITTEN*
```

Reply:
fixed thanks